terraform/20-app/ecs.jobs.crawl-front-end.tf

```diff
@@ -0,0 +1,11 @@
+resource "local_sensitive_file" "ecs_job_crawl_front_end" {
```

Comment:
Made a new ECS job because the hydrate-frontend-cache job pulls in the larger more expensive utility worker